google/api/routing.proto

```diff
@@ -0,0 +1,347 @@
+/*
+ This proto is here only until cl/378772749 is merged
```

Comment:
cl/397839221 is merged, and `rounting.proto` is already present in the repo. Closing.

Reply:
Actually, I don't have permissions to close this. @viacheslav-rostovtsev will you close this?